CMakeLists.txt

```diff
@@ -47,7 +49,20 @@ include(GNUInstallDirs)
 include(FetchContent)
 include(cmake/code_coverage.cmake)
 include(cmake/sanitizer.cmake)
-include(cmake/gcc_analyze.cmake)
+include(cmake/clear_variable.cmake)
+
+# Linting and Static Analyze
+if(CPR_ENABLE_LINTING AND CPR_STATIC_ANALYZE)
+    message(FATAL_ERROR "Linting and the GCC static analysis can not be enabled at the same time! Disable either CPR_STATIC_ANALYZE or CPR_ENABLE_LINTING.")
+endif()
```

Comment:
I think CPR should be fine with clang-tidy and gcc-analyze running in parallel. (It is only a RAM problem for bigger Projects)